src/main/java/com/sourcegraph/telemetry/GraphQlLogger.java

```diff
@@ -14,7 +15,9 @@ public class GraphQlLogger {
   public static CompletableFuture<Boolean> logInstallEvent(@NotNull Project project) {
     CodyApplicationSettings codyApplicationSettings = CodyApplicationSettings.getInstance();
     if (codyApplicationSettings.getAnonymousUserId() != null && !project.isDisposed()) {
-      var event = createEvent(ConfigUtil.getServerPath(project), "CodyInstalled", new JsonObject());
+      var event =
+          createEvent(
+              project, ConfigUtil.getServerPath(project), "CodyInstalled", new JsonObject());
```

Comment:
btw, can we have an enum class for the event name? 🙏 it would be easier to navigate b/w then and see which are already in the code in a one place